README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
+* Mocha test framework.
+
+### Installation
+1. Install docker on your machine: https://www.docker.com/
+
+2. In your root directory, create a `wdio.conf.js` file that inherits from Terra Toolkit's base config.
+
+```js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  // TOOD: Custom wdio config goes here. See: http://webdriver.io/guide/testrunner/configurationfile.html
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+### Selenium Docker
+Selenium docker is provided as a convenience to make selenium testing easier and more stable. Running selenium in a container ensures a consistent testing environment across testing environments which is critical for visual regression testing.
+
+#### Options
+
+Under the key `seleniumDocker` in your wdio.conf.js you can pass a configuration object with the following structure:
+
+* **cidfile** - The name of the docker cidfile used to manage the docker instance during tests. Defaults to '.docker_selenium_id',
+* **enabled** - Flat to disable selenium docker; useful for CI environments which can startup the docker instance outside of test runs. Defaults to true
+* **cleanup** - Destroy the docker container after the test run. Defaults to false.
+* **image** - The docker image to use for test runs. Defaults to `selenium/standalone-chrome` or `selenium/standalone-firefox` based on browser capabilities specified in config.
+
+#### Example
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  seleniumDocker: {
+    // Disable if running in Travis
+    enabled: !process.env.TRAVIS,
+  },
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+
+### Accessibility Tests
+Terra toolkit automatically includes a wdio-axe-service which enhances an WebdriverIO instance with commands for accessibility testing using the [Axe](https://github.com/dequelabs/axe-core) utility.
+
+#### Options
+
+Under the key `axe` in your wdio.conf.js you can pass a configuration object with the following structure:
+
+* **inject** - True if the axe script should be injected by the test running. Disable if axe is already included in the test files which slightly speed up runs. Defaults to true.
+
+#### Example
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  axe: {
+    // Don't inject axe script, its included in test files
+    inject: false,
+  },
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+
+#### Usage
+
+`browser.axe([{options}]);`
+
+The following options are available:
+
+* **viewports**:
```

Comment:
In you're tests you specify viewports as 'tiny', etc which maps to this {width, height} obj in the terra service. I would maybe update this to mention when using the terraService this is an option

Reply:
Or mention when used with terraService use viewport helper & link to section

Reply:
I'll call it out in the example documentation below